File: JAVA/3274. Check if Two Chessboard Squares Have the Same Color.java
class Solution {
    public boolean checkTwoChessboards(String c1, String c2) {
        int a=(int)(c1.charAt(0));
        int b=(int)(c2.charAt(0));
        a+=(c1.charAt(1)-'0');
        b+=(c2.charAt(1)-'0');
        return (a%2==0)==(b%2==0);
    }
}

File: C++/3274. Check if Two Chessboard Squares Have the Same Color.cpp
class Solution {
public:
    bool checkTwoChessboards(string c1, string c2) {
        int a=(int)(c1[0]);
        int b=(int)(c2[0]);
        a+=(c1[1]-'0');
        b+=(c2[1]-'0');
        return (a%2==0)==(b%2==0);
    }
};